packages/terra-alert/src/terra-dev-site/doc/alert/Alert.doc.jsx

```diff
@@ -9,18 +9,31 @@ import AlertSrc from '!raw-loader!../../../../src/Alert';
 
 // Example Files
 import AlertExample from '../example/AlertExample';
+import AlertExampleSrc from '!raw-loader!../example/AlertExample';
```

Comment:
We need to update each of the Alert example to remove the custom src code that was hard-coded...For example: https://github.com/cerner/terra-core/blob/master/packages/terra-alert/src/terra-dev-site/doc/example/ActionExample.jsx#L22-L40

Reply:
Updated: https://github.com/cerner/terra-core/pull/2130/commits/ac8e9c25a6474a23ee6a9e73ac8e43e80743f4fc